server/build/docker-compose.common.yml

```diff
@@ -1,7 +1,7 @@
 version: '2.4'
 services:
   mysql:
-    image: "mysql/mysql-server:5.7.12"
+    image: "mysql/mysql-server:8.0.31"
```

Comment:
8.0.32 was the latest in aurora mysql. Happy to bump to that.

Reply:
Yeah, let's keep it the same. I also asked because for v5 I recall we were taking a bit of a different approach, supporting the earliest compatible version. Not opposed to using the latest here, but trying to understand if the plan is to stick with this version for the long term or to gradually upgrade it in the future as new releases come out?

Reply:
I think it will be a maintenance burden to keep updating minor versions, _and_ keep them in sync with the load test tool. I'm leaning towards just this single version.

Reply:
Sounds good :+1: